lasagne/layers/cuda_convnet.py

```diff
@@ -617,7 +622,7 @@ class NINLayer_c01b(Layer):
         Any additional keyword arguments are passed to the `Layer` superclass.
     """
     def __init__(self, incoming, num_units, untie_biases=False,
-                 W=init.GlorotUniform(c01b=True), b=init.Constant(0.),
+                 W=init.GlorotUniform(), b=init.Constant(0.),
```

Comment:
whoops! I didn't expect this to bite us so soon, a better solution than #259 may be in order.